### DISCUSSION TITLE
interpreting motif deviation scores

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1109] [Questions / Documentation]
Hello,

Can you please help me understand if I'm interpreting the motif deviations correctly? I found these TFs through "positive regulator" analysis using the gene expression matrix as the correlate to my motif matrix. 
![Screen Shot 2021-10-26 at 2 23 47 PM](https://user-images.githubusercontent.com/92815743/138963979-fc272b1f-8b89-405a-b4cf-3fbfdecbfee1.png)

Let me focus on NEUROD1. To me, this data says the genome-wide motifs that NEUROD1 is expected to bind to are more accessible in "RNA-1" relative to the other clusters. Is that accurate?

Thanks

**Comment by rcorces** [comment]
yes
